tests/requirements/test_hal_watchdog.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-# import logging
+import logging
```

Comment:
Necessary?

Reply:
good to have, leave it for now. Cargo doesn't care about logs, we're checking exit code anyway.